.github/workflows/main.yml

```
@@ -0,0 +1,32 @@
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  evaluator_job:
+    runs-on: ubuntu-18.04
+    name: Evaluator job
+    steps:
+      - name: Wait for build to succeed
+        uses: betrybe/action-wait-for-check@v2
+        id: wait-for-build
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: codeclimate
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+      - uses: actions/checkout@v2
+        if: steps.wait-for-build.outputs.conclusion == 'success'
+      - name: Evaluator step
+        id: test
+        uses: betrybe/jest-evaluator-action@v5
+        if: steps.wait-for-build.outputs.conclusion == 'success'
+        with:
+          repository-test-name: 'betrybe/sd-0x-a-cli-of-ice-and-fire-tests'
+          repository-test-branch: 'master'
+      - name: Store evaluation step
+        uses: betrybe/store-evaluation-action@v2
+        if: steps.wait-for-build.outputs.conclusion == 'success'
+        with:
+          evaluation-data: ${{ steps.test.outputs.result }}
+          environment: staging
+          pr-number: ${{ github.event.number }}
```

Adiciona uma nova linha ao final do arquivo, pls
```suggestion
          pr-number: ${{ github.event.number }}

```